### DISCUSSION TITLE
PHP 8.1 Support

### DISCUSSION THREAD

**Discussion by xmav** [php-amqplib/php-amqplib#950] [Q&A] [answered]
Hi!

My questions same as in #901 but for PHP8.1. Is there any plans for release with PHP 8.1 support? Will it be 3.1.0 ?

Thanks in advance

**Comment by ramunasd** [comment, marked as answer]
Yes, PHP8.1 support was released in version [3.1](https://github.com/php-amqplib/php-amqplib/blob/master/CHANGELOG.md#v310-2021-10-22)
